sonar-ruby-plugin/pom.xml

```diff
@@ -159,7 +167,125 @@
                             <filters>
                                 <filter>
                                     <artifact>*:*</artifact>
+                                    <!-- not listed items are implicitly excluded -->
+                                    <includes>
+                                        <include>org/sonar/l10n/**</include>
+                                        <include>org/sonarsource/**</include>
+                                        <include>whitequark_parser_init.rb</include>
+
+                                        <!-- from rubygems:racc -->
+                                        <include>racc-${rubygems.racc.version}-java/lib/**</include>
+
+                                        <!-- from rubygems:ast -->
+                                        <include>ast-${rubygems.ast.version}/lib/**</include>
+
+                                        <!-- from rubygems:parser -->
+                                        <include>parser-${rubygems.parser.version}/lib/**</include>
+
+                                        <!-- from org.jruby:jruby-complete -->
+                                        <include>com/headius/**</include>
+                                        <include>jnr/constants/**</include>
+                                        <include>jnr/enxio/**</include>
+                                        <include>jnr/ffi/mapper/FunctionMapper.class</include>
+                                        <include>jnr/ffi/Pointer.class</include>
+                                        <include>jnr/ffi/Struct.class</include>
+                                        <include>jnr/posix/**</include>
+                                        <include>jnr/unixsocket/**</include>
+                                        <include>jruby/**</include>
+                                        <include>META-INF/**</include>
+                                        <include>org/jcodings/**</include>
+                                        <include>org/joda/**</include>
+                                        <include>org/joni/**</include>
+                                        <include>org/jruby/**</include>
+                                        <include>tables/CR_Digit.bin</include>
+                                        <include>tables/CR_Space.bin</include>
+                                        <include>tables/Transcoder_Utf1632_ByteArray.bin</include>
+                                        <include>tables/Transcoder_Utf1632_WordArray.bin</include>
```

Comment:
It seems that we should also include "/tables/CR_Word.bin". I got the following error when analyzing [this file](https://github.com/octobox/octobox/blob/master/lib/search_parser.rb):
```
ERROR: Unable to parse file: file:///Users/pierre-yvesnicolas/test/ruby/octobox/lib/search_parser.rb. Parse error at position 5:24
ERROR: (SyntaxError) entry: /tables/CR_Word.bin not found: /(\w+):[\ \u3000]?([\w\p{Han}\p{Katakana}\p{Hiragana}\p{Hangul}\u30FC\.\-,\/]+|(["'])(\\?.)*?\3)/
```